synapse/storage/schema/delta/54/account_validity.sql

```diff
@@ -16,5 +16,10 @@
 -- Track what users are in public rooms.
 CREATE TABLE IF NOT EXISTS account_validity (
     user_id TEXT PRIMARY KEY,
-    expiration_ts_ms BIGINT NOT NULL
+    expiration_ts_ms BIGINT NOT NULL,
+    email_sent BOOLEAN NOT NULL,
+    renewal_token TEXT
 );
+
+CREATE INDEX account_validity_email_sent_idx ON account_validity(email_sent, expiration_ts_ms)
+CREATE UNIQUE INDEX account_validity_renewal_string_idx ON account_validity(renewal_token)
```

Comment:
You need to put this in a separate file, otherwise these changes won't be picked up if we deployed any servers from develop after the first PR. Stick the following in a `account_validity_expiry.sql` and it should be fine (as the files get run in alphabetical order)

```sql
ALTER TABLE account_validity ADD COLUMN expiration_ts_ms BIGINT NOT NULL;
ALTER TABLE account_validity ADD COLUMN email_sent BOOLEAN NOT NULL;
ALTER TABLE account_validity ADD COLUMN renewal_token TEXT;

CREATE INDEX account_validity_email_sent_idx ON account_validity(email_sent, expiration_ts_ms)
CREATE UNIQUE INDEX account_validity_renewal_string_idx ON account_validity(renewal_token)
```

Reply:
ftr the `expiration_ts_ms` (first one) is not necessary since this column was already there since #5026